tests/Composer/app/index.php

```diff
@@ -39,4 +40,11 @@
 Logger::set(new ErrorLogLogger('debug'));
 Logger::get()->debug('some-debug-message');
 
+/*
+ * Accessing tracer version, which was loaded from a physical file until 0.48.3.
+ */
```

Comment:
Style nit suggestion:

```suggestion
// Accessing tracer version, which was loaded from a physical file until 0.48.3.
```